tests/Unit/CrudPanel/CrudPanelDeleteTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Backpack\CRUD\Tests\Unit\CrudPanel;
+
+use Illuminate\Support\Facades\DB;
+use Backpack\CRUD\Tests\Unit\Models\Article;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+class CrudPanelDeleteTest extends BaseDBCrudPanelTest
+{
+    public function testDelete()
+    {
+        $this->markTestIncomplete('Not correctly implemented');
+
+        $this->crudPanel->setModel(Article::class);
+        $article = Article::find(1);
+
+        $wasDeleted = $this->crudPanel->delete($article->id);
+
+        // TODO: the delete method should not convert the returned result to a string
+        $deletedArticle = Article::find(1);
+        $this->assertTrue($wasDeleted);
+        $this->assertNull($deletedArticle);
+    }
+
+    public function testDeleteUnknown()
+    {
+        $this->markTestIncomplete('Not correctly implemented');
+
+        $this->setExpectedException(ModelNotFoundException::class);
+
+        $this->crudPanel->setModel(Article::class);
+        $unknownId = DB::getPdo()->lastInsertId() + 1;
+
+        // TODO: if the delete method is called with an unknown id, it should return a model not found exception. change
```

Comment:
That will probably cause those not expecting an exception on delete to fall over; proceed with caution with the (not yet filed?) upstream bug lest we break things unexpectedly.